src/network/networkd-bridge-mdb.h

```diff
@@ -10,10 +10,21 @@
 typedef struct Link Link;
 typedef struct Network Network;
 
+typedef enum EntryType {
```

Comment:
"EntryType" is *very* generic a name. Maybe call this BridgeMDBEntryType.

Reply:
Agree, fixed.